src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -769,6 +774,19 @@ private function registerRequestConfiguration(array $config, ContainerBuilder $c
         }
     }
 
+    private function registerEncryptedSecretsConfiguration(array $config, ContainerBuilder $container)
+    {
+        $secretsFile = $config['encrypted_secrets']['secrets_file'];
+        $publicKeyFile = $config['encrypted_secrets']['public_key_file'];
+        $privateKeyFile = $config['encrypted_secrets']['private_key_file'];
+        $secretsHandler = new JweHandler($secretsFile, $publicKeyFile, $privateKeyFile);
+        //TODO: this will only validate presence/size of files, make sure the configuration can decrypt values before runtime
+
+        $container->fileExists($secretsFile);
```

Comment:
I don't think this is needed. We actually *don't* need to validate (shouldn't) that this file exists while building the container. In theory, we should be able to build the container without this file even existing. Then, on production, we "ship" the secrets file and the container starts using it. Also, during dev, we don't need the container to be rebuilt when this file changes, right? If this file changes, the dumped container would just instantly start using the new value.

Reply:
this was also to ensure that things would blow up before run-time, i will remove